src/bfcli/main.c

```diff
@@ -158,6 +158,56 @@ int _bf_do_ruleset_get(int argc, char *argv[])
 #define BFC_COMMAND_NAME_LEN 32
 static char _bfc_command_name[BFC_COMMAND_NAME_LEN];
 
+static const char help_message[] =
+    "bfcli - Command line interface for bpfilter daemon\n\n"
+    "Usage: bfcli [OBJECT] [ACTION] [OPTIONS]\n\n"
+    "Objects and Actions:\n"
+    "  ruleset\n"
+    "    set     - Define a new ruleset and replace all existing chains\n"
+    "              Options:\n"
+    "                --file FILE     - Read ruleset from file\n"
+    "                --str RULESET   - Read ruleset from command line\n"
+    "    get     - Print current ruleset with counters\n"
+    "    flush   - Remove all chains and rules\n\n"
+    "  chain\n"
+    "    set     - Generate and load a chain into kernel\n"
+    "              Options:\n"
+    "                --from-str CHAIN - Read chain from command line\n"
+    "                --from-file FILE - Read chain from file\n"
+    "                --name NAME      - specify which one to send to the daemon if --from-str or --from-file provide multiple chains.\n"
```

Comment:
`specify` -> `Specify`

Unsure: `provide` -> `provides`?

Reply:
I think "provides" is right since they are singular